<launch>
  <!--
  	This pushes the local PYTHONPATH into the launch file, so that the NaoQI API is found.
    You need to add the Nao's API dir to your PYTHONPATH so that the modules are found.
  -->
  <env name="PYTHONPATH" value="$(env PYTHONPATH)" />

  <arg name="nao_ip" default="$(optenv NAO_IP 10.152.246.59)" />
  <arg name="nao_port" default="$(optenv NAO_PORT 9559)" />

  <rosparam file="$(find clever_nao)/launch/configs/data.yaml" command="load"/>

  <!-- launch python server -->
  <!-- <node pkg="clever_nao" type="face_server.py" name="face_server" required="true" args="$(arg nao_ip) $(arg nao_port)" output="screen"/> -->
  <node pkg="clever_nao" type="hear_server.py" name="hear_server" required="true" args="$(arg nao_ip) $(arg nao_port)" output="screen"/>
  <!-- <node pkg="clever_nao" type="speak_server.py" name="speak_server" required="true" args="$(arg nao_ip) $(arg nao_port)" output="screen"/> -->
  <!-- <node pkg="clever_nao" type="walk_server.py" name="walk_server" required="true" args="$(arg nao_ip) $(arg nao_port)" output="screen"/> -->

  <!-- launch C++ server -->
  <!-- <node pkg="clever_nao" type="qrcode_server" name="qrcode_server" output="screen"/> -->


</launch>
